websites/G/Granblue Fantasy Wiki/presence.ts

```diff
@@ -0,0 +1,50 @@
+const presence = new Presence({
+    clientId: "914354609370329098"
+  }),
+  browsingTimestamp = Math.floor(Date.now() / 1000);
+
+
+presence.on("UpdateData", () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "vyrnball"
+  };
+  presenceData.startTimestamp = browsingTimestamp;

```

Comment:
Move this into the PresenceData object above

Reply:
Fixed